src/python/WMCore/ReqMgr/Utils/Validation.py

```diff
@@ -159,13 +159,21 @@ def validate_resubmission_create_args(request_args, config, reqmgr_db_service, *
         # ACDC of ACDC, we can't validate this case
         # simply copy the whole original dictionary over and accept all args
         createArgs = originalArgs
+        # TODO: This code can cause the problem when ACDC is created before this update which
+        # TODO: doesn't contain "TopLevelOriginalRequestType" so only uncomment this all the acdc are created after this updated
+        #request_args["TopLevelOriginalRequestType"] = originalArgs["TopLevelOriginalRequestType"]
+
+        # TODO: Remove this line when all the acdc are created after this patch is applied and uncomment above line.
+        # WARNING: This doesn't solve the problem when when this is ACDC of ACDC or more. ( > 2 level ACDC)
+        request_args["TopLevelOriginalRequestType"] = originalArgs.get("TopLevelOriginalRequestType") or originalArgs["OriginalRequestType"]
```

Comment:
I don't get your comment, if it's an ACDC of an ACDC, `TopLevelOriginalRequestType` value will be the original RequestType, thus `TopLevelOriginalRequestType` will always have the original request type, no matter which ACDC level it is. Am I missing something (besides the first ACDC being created before this patch)?

Reply:
Yes exactly, it doesn't fix if the first ACDC is created before the patch and we have to make the ACDC of that ACDC.